x-pack/solutions/security/plugins/security_solution/public/data_view_manager/components/data_view_picker/index.tsx

```diff
@@ -96,14 +102,20 @@ export const DataViewPicker = memo(({ scope, onClosePopover, disabled }: DataVie
   /**
    * Selects data view again. After changes are made to the data view, this results in cache invalidation and will force the reload everywhere.
    */
-  const handleDataViewModified = useCallback(
-    (updatedDataView: DataView) => {
-      selectDataView({ id: updatedDataView.id, scope: [scope] });
-    },
-    [scope, selectDataView]
+  const handleDataViewModified = useMemo(
+    () =>
+      canEditDataView
```

Comment:
It could be _cleaner_ to do the check where the fn is passed as the actual prop, but that's a personal preference thing 🤷🏾‍♂️